spec/api-browser-window-spec.js

```diff
@@ -1923,14 +1922,30 @@ describe('BrowserWindow module', () => {
 
       it('validates process APIs access in sandboxed renderer', (done) => {
         ipcMain.once('answer', function (event, test) {
-          assert.strictEqual(test.arch, remote.process.arch)
-          assert.strictEqual(test.platform, remote.process.platform)
-          assert.deepStrictEqual(...resolveGetters(test.env, remote.process.env))
-          assert.strictEqual(test.execPath, remote.process.helperExecPath)
-          assert.strictEqual(test.sandboxed, true)
-          assert.strictEqual(test.type, 'renderer')
-          assert.strictEqual(test.version, remote.process.version)
-          assert.deepStrictEqual(test.versions, remote.process.versions)
+          expect(test.hasCrash).to.be.true()
+          expect(test.hasHang).to.be.true()
+          expect(test.heapStatistics).to.be.an('object')
+          expect(test.processMemoryInfo).to.be.an('object')
+          expect(test.systemVersion).to.be.a('string')
+          expect(test.cpuUsage).to.be.an('object')
+          expect(test.ioCounters).to.be.an('object')
+          expect(test.arch).to.equal(remote.process.arch)
+          expect(test.platform).to.equal(remote.process.platform)
+          expect(test.env).to.deep.equal(remote.process.env)
+          expect(test.execPath).to.equal(remote.process.helperExecPath)
+          expect(test.sandboxed).to.be.true()
+          expect(test.type).to.equal('renderer')
+          expect(test.version).to.equal(remote.process.version)
+          expect(test.versions).to.deep.equal(remote.process.versions)
+
+          if (process.platform === 'linux' && test.osSandbox) {
```

Comment:
This behavior does not appear to be documented here:

https://github.com/electron/electron/blob/master/docs/api/process.md#processgetsystemmemoryinfo

I wouldn't expect it to fail in sandbox?

Reply:
I can fix the docs in a follow up PR